### PR TITLE
more linting

### PR DESCRIPTION
**Pull request by JensTimmerman**
added a few more lint checks that check things related to our coding styleguides, 
e.g. line too long and use of explicit octal numbers.

just here for reference, can be merged in a while once we feel comfortable doing more cleanup

### CODE REVIEW

File: lib/vsc/install/commontest.py [hunk 1]
```diff
@@ -103,9 +103,22 @@ class CommonTest(TestCase):
...
+        # 'logging-not-lazy',
+        # will stop working in python3
+        'unpacking-in-except', 'redefine-in-handler',  # except A, B -> except (A, B)
+        'indexing-exception',  # indexing exceptions doesn't work in python3, use Exc.args[index] instead (but why?)
+        'raising-string',  # don't raise strings, raise objects extending Exception
+        'old-octal-literal',  # use 0o700 instead of 0700
+        'import-star-module-level',  # Import * only allowed at module level
+        'old-ne-operator',  # don't use <> as not equal operator, use !=
+        'backtick',  # don't use `variable` to turn a variable in a string, use the str() function
+        'old-raise-syntax',  # sed when the alternate raise syntax raise foo, bar is used instead of raise foo(bar) .
```
Comment (wpoely86) [inline lib/vsc/install/commontest.py]: can we put this list in alphabetical order?

File: lib/vsc/install/shared_setup.py [hunk 2]
```diff
@@ -229,7 +229,10 @@ def _fvs(msg=None):
 
     subclasses = parent.__subclasses__()
     if len(subclasses) > 1:
-        log.warn("%sMore than one %s subclass found (%s), returning the first one" % (msg, pname, [x.__name__ for x in subclasses]))
+        log.warn("%sMore than one %s subclass found (%s), returning the first one",
+                 msg,
+                 pname,
+                 [x.__name__ for x in subclasses])
```
Comment (wpoely86) [inline lib/vsc/install/shared_setup.py]: put these last three on 1 line?

File: lib/vsc/install/commontest.py [hunk 3]
```diff
@@ -103,9 +103,22 @@ class CommonTest(TestCase):
         'F811',  # redefinition of unused name
         'unused-import',
         'syntax-error',
-        'unpacking-in-except', 'redefine-in-handler', # except A, B -> except (A, B)
-        #'protected-access',
-        #'logging-not-lazy',
+        'duplicate-key',  # when a key appears twice in a dict definition
+        'E501',  # 'line too long'when a line is longer then 120 chars
```
Comment (wpoely86) [inline lib/vsc/install/commontest.py]: where does it get set to 120 chars? It's 72 by default?

**Comment by wpoely86** [comment]
@JensTimmerman version bump is missing!

File: lib/vsc/install/commontest.py (hunk 3 above)
Comment (JensTimmerman) [inline lib/vsc/install/commontest.py]: further down in this file, we tell prospector that lines are 120chars

File: lib/vsc/install/shared_setup.py (hunk 2 above)
Comment (JensTimmerman) [inline lib/vsc/install/shared_setup.py]: is there a linting rule that can help detect when I need to do this?

**Pull request merged**